### PR TITLE
Fixup for `Axes.expand` in the early return case

### PR DESCRIPTION
**Pull request by etpinard**
Fixup for commit https://github.com/plotly/plotly.js/pull/1355/commits/cebc31c72f20cd0e1426e8d9258f9e09e269817a part of the multiple range slider PR https://github.com/plotly/plotly.js/pull/1355

### CODE REVIEW

File: src/plots/cartesian/axes.js [hunk 1]
```diff
@@ -378,7 +378,11 @@ axes.saveRangeInitial = function(gd, overwrite) {
 //      tozero: (boolean) make sure to include zero if axis is linear,
 //          and make it a tight bound if possible
 axes.expand = function(ax, data, options) {
-    var needsAutorange = (ax.autorange || Lib.nestedProperty(ax, 'rangeslider.autorange'));
+    var needsAutorange = (
+        ax.autorange ||
+        !!Lib.nestedProperty(ax, 'rangeslider.autorange').get()
```
Comment (etpinard) [inline src/plots/cartesian/axes.js]: Even if `ax.rangeslider` is undefined, `Lib.nestedProperty(ax, 'rangeslider.autorange')` return an object (which is of course truthy) - which made `needsAutorange` always be truthy, removing the early return :racehorse: path.

File: test/jasmine/tests/axes_test.js [hunk 2]
```diff
@@ -1264,11 +1264,11 @@ describe('Test axes', function() {
         // way of getting a new clean copy each time.
         function getDefaultAx() {
             return {
+                autorange: true,
                 c2l: Number,
                 type: 'linear',
                 _length: 100,
-                _m: 1,
-                _needsExpand: true
```
Comment (etpinard) [inline test/jasmine/tests/axes_test.js]: That's an artefact that of the old range slider + autorange implementation that should've been taken out in https://github.com/plotly/plotly.js/pull/1355/commits/cebc31c72f20cd0e1426e8d9258f9e09e269817a 

My mistake.

File: src/plots/cartesian/axes.js (hunk 1 above)
Comment (alexcjohnson) [inline src/plots/cartesian/axes.js]: haha I've made exactly that mistake more than once...

**Comment by alexcjohnson** [comment]
💃

**Pull request merged**